### PR TITLE
Add collection equivalent assertion for NUnit

### PR DESCRIPTION
**Pull request by lucasteles**

### CODE REVIEW

File: tests/FsUnit.NUnit.Test/equalTests.fs [hunk 1]
```diff
@@ -21,6 +21,22 @@ type ``equal Tests`` ()=
...
+     ``collection type should not equal equivalent if is not in same order`` ()=
+        [1;2;3] |> should  not' (equal [3;2;1])
+
+    [<Test>] member test.
+     ``collection type should equivalent collection `` ()=
+        [1..10] |> should equivalent [1..10]
+
+    [<Test>] member test.
+     ``collection type should equal equivalent independent of order`` ()=
+        [1;2;3] |> should  equivalent [3;2;1]
```
Comment (CaptnCodr) [inline tests/FsUnit.NUnit.Test/equalTests.fs]: Remove double space between `should` and `equivalent`.

File: tests/FsUnit.NUnit.Test/equalTests.fs [hunk 2]
```diff
@@ -21,6 +21,22 @@ type ``equal Tests`` ()=
      ``value type should equal equivalent value`` ()=
         1 |> should equal 1
 
+    [<Test>] member test.
+     ``collection type should equal collection`` ()=
+        [1..10] |> should equal [1..10]
+
+    [<Test>] member test.
+     ``collection type should not equal equivalent if is not in same order`` ()=
+        [1;2;3] |> should  not' (equal [3;2;1])
```
Comment (CaptnCodr) [inline tests/FsUnit.NUnit.Test/equalTests.fs]: Remove double space between `should` and `not`.

**Comment by lucasteles** [comment]
I removed the spaces

File: tests/FsUnit.NUnit.Test/equalTests.fs [hunk 3]
```diff
@@ -21,6 +21,22 @@ type ``equal Tests`` ()=
...
+    [<Test>] member test.
+     ``collection type should equal collection`` ()=
+        [1..10] |> should equal [1..10]
+
+    [<Test>] member test.
+     ``collection type should not equal equivalent if is not in same order`` ()=
+        [1;2;3] |> should not' (equal [3;2;1])
+
+    [<Test>] member test.
+     ``collection type should equivalent collection `` ()=
```
Comment (CaptnCodr) [inline tests/FsUnit.NUnit.Test/equalTests.fs]: Here is a space too much as well after second `collection`.

**Comment by lucasteles** [comment]
done

**Comment by sergey-tihon** [comment]
@lucasteles can you please add implementation for XUnit and MsTest  as well?
and add new operator to https://github.com/fsprojects/FsUnit/blob/master/docs/content/operators.md

**Comment by lucasteles** [comment]
@sergey-tihon  yes, i can,

I didn't have time the last week, but I think I could work on that this week

**Comment by sergey-tihon** [comment]
@lucasteles thank you for all your effort, but I close it in favor of #145

**Pull request closed without merging**